src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java

```diff
@@ -1134,15 +1143,19 @@ public final void writeTo(StringBuilder builder) {
     }
 
 
-    private void sendMetric(final Message message) {
-        send(message);
-        this.telemetry.incrMetricsSent(1);
+    private boolean sendMetric(final Message message) {
```

Comment:
Can this method be removed since now it's just a passthrough to `send`?

Reply:
I can check, in any case, I'd rather clean that up in a separate PR? Being a private method it's probably fine, but it might imply some refactoring. I can't remember off the top of my head why we kept it around.